packages/modules/providers/auth-emailpass/src/services/emailpass.ts

```diff
@@ -43,15 +43,15 @@ export class EmailPassAuthService extends AbstractAuthModuleProvider {
   }
 
   async update(
-    data: { email: string; password: string },
+    data: { password: string; entity_id: string },
```

Comment:
We currently only allow password updates, this could possibly be extended in the future.